util.go

```diff
@@ -117,3 +117,86 @@ func trDelete(s, substr string) string {
 	}
 	return ret.String()
 }
+
+// createMediaMessage returns a `Chattable' (which can be sent with bot.Send)
+// targeted at `destination', made with the media received in the message passed
+// as parameter, if there is any media in this message.
+func createMediaMessage(message *tgbotapi.Message, destination int64, markup *tgbotapi.InlineKeyboardMarkup) (tgbotapi.Chattable, error) {
+	if message == nil {
+		return nil, fmt.Errorf("message is nil")
```

Comment:
I'd expand the error message to something like "cannot process empty message" or something more descriptive.

Reply:
Done. Used your wording.